tests/config_helper.py

```diff
@@ -0,0 +1,2 @@
+def input_example_and_signature_on():
```

Comment:
Is this necessary? I see we're passing it to most of the `mlflow.*.autolog()` calls throughout our test cases. In general, I think it makes sense to test the default set of arguments (i.e. leave the arguments unspecified) and only mutate the values when we're testing specific behavior. For example, `input_example_and_signature_on` doesn't seem essential for test cases like `test_lightgbm_autolog.py#test_lgb_autolog_ends_auto_created_run`.

Reply:
hmm fair enough. I was thinking more along the lines of avoiding to break the test cases, but I guess for those that it did break I can pass the values that the test is expecting.